requirements-build.txt

```diff
@@ -119,37 +121,35 @@ wheel==0.45.1
     #   tzdata
 
 # The following packages are considered to be unsafe in a requirements file:
-setuptools==69.0.2 ; python_version >= "3.12" and python_version < "3.14"
-    # via
-    #   ansible-runner
-    #   setuptools-scm
-setuptools==75.6.0 ; python_version >= "3.12" and python_version < "3.14"
+setuptools==75.8.0 ; python_version >= "3.12" and python_version < "3.14"
     # via
     #   ansible
     #   ansible-core
     #   bcrypt
     #   cachetools
-    #   calver
     #   cffi
-    #   charset-normalizer
     #   click-repl
+    #   cryptography
     #   django
     #   django-cors-headers
     #   gunicorn
     #   markupsafe
-    #   maturin
     #   pathspec
+    #   pluggy
     #   psycopg
     #   psycopg-c
     #   pyasn1
     #   pyasn1-modules
+    #   pydantic
     #   python-daemon
+    #   python-dateutil
     #   pyyaml
     #   resolvelib
     #   setuptools-scm
+    #   trove-classifiers
     #   tzdata
     #   whitenoise
-setuptools==75.7.0 ; python_version >= "3.12" and python_version < "3.14"
+setuptools==69.0.2 ; python_version >= "3.12" and python_version < "3.14"
```

Comment:
ugh, I need to open the PR on pybuild-deps that fixes this mess with annoations and ordering

Reply:
:sadnod: